storagenode/storagenodedb/pieceinfo.go

```diff
@@ -102,3 +104,31 @@ func (db *pieceinfo) SpaceUsed(ctx context.Context) (int64, error) {
 	}
 	return *sum, err
 }
+
+// GetExpired gets orders that are expired and were created before some time
+func (db *pieceinfo) GetExpired(ctx context.Context, expiredAt time.Time) (info []pieces.Info, err error) {
```

Comment:
Don't return `pieces.Info` if objects will be only partially filled (only SatelliteID and PieceID)

Reply:
the unused/filled field will be null or zero. Otherwise, need to define other structure and slice of it.. .adding extra line of code, that could be avoided.

Reply:
But someone who will use this method will think that whole struct is complete (wont look into method body), will make wrong assumption and will introduce an issue.

Reply:
done